src/util/c_types.h

```diff
@@ -32,7 +32,13 @@ unsignedbv_typet char32_t_type();
 bitvector_typet float_type();
 bitvector_typet double_type();
 bitvector_typet long_double_type();
+bitvector_typet gcc_float16_type();
+bitvector_typet gcc_float32_type();
+bitvector_typet gcc_float32x_type();
+bitvector_typet gcc_float64_type();
+bitvector_typet gcc_float64x_type();
 bitvector_typet gcc_float128_type();
+bitvector_typet gcc_float128x_type();
```

Comment:
We should really start splitting this file into "platform_types" (which belongs in util/) and the language-specific "c_types".

Reply:
These are even more specific, and likely should only live in ansi-c/; I'll move.

Reply:
I think it was my (wrong-)doing to move `c_types.{h,cpp}` from `ansi-c` to `util`. What should have happened is: move some of the contents of `ansi-c/c_types.{h,cpp}` to `util/platform_types.{h,cpp}`. It might be about time to fix this.